.github/workflows/promote-reusable-workflow.yml

```diff
@@ -0,0 +1,43 @@
+# Reusable workflow for promote jobs.
+
+on:
+  workflow_call:
+    inputs:
+      channel-name:
+        description: 'Human readable name of the channel to promote'
+        required: true
+        type: string
+      ts-file:
+        description: 'Machine name of the TS file to execute, under workflows/'
+        required: true
+        type: string
+      amp-version:
+        description: 'AMP version number (13 digits). Not all promote scenarios require an explicit version.'
+        required: false
+        type: number
+
+jobs:
+  promote:
+    if: github.repository == 'ampproject/cdn-configuration'
+    name: Promote ${{ github.event.inputs.channel-name }} Channel
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
```

Comment:
Does this need to be specified?

Reply:
Eh, not necessarily but it would make things faster once this repo has thousands of commits (e.g., if we commit at least once every weekday, within 2 years there'll be ~500 commit in the repo)

Reply:
This action already fetches a single commit by default.

Reply:
Oh I didn't realize that. Done 9330fe8